### DISCUSSION TITLE
[PSD1] Exercise 1.22

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#115] [Esercizi (A-L)]
**Exercise 1.22** Convert the following two’s complement binary numbers to decimal

(a) $1110_2$
(b) $100011_2$
(c) $01001110_2$
(d) $10110101_2$

**Comment by Elia-Belli** [comment]
(a) $1110_2 = (-8+4+2) = -2_{10}$
(b) $100011_2 = (-32+2+1) = -29_{10}$
(c) $01001110_2 = (64+2+4+8) = 78_{10}$
(d) $10110101_2 = (-128+32+16+4+1) = -75_{10}$

**Comment by niccolo-fato** [comment]
![photo_2024-11-15_16-21-43](https://github.com/user-attachments/assets/50ccf2bd-67f0-44b9-9746-839c54b8c129)
